### PR TITLE
merging the elements of the contact page

### PR DESCRIPTION
**Pull request by Vjerem**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Header from './components/Header';
 import './reset.css';
 import './App.css';
+import Contact from './components/Contact';
```
Comment (KeerhemFR) [inline src/App.jsx]: Must be above, the css sheets arrive after the components

File: src/components/Contact/index.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import SContact from './SContact';
```
Comment (KeerhemFR) [inline src/components/Contact/index.jsx]: Style sheets must come after components

File: src/components/Contact/SContact.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const SContact = styled.div`
+  width: 100%;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-evenly;
+  align-items: center;
+
+  & & <* {
```
Comment (KeerhemFR) [inline src/components/Contact/SContact.jsx]: Don't understand that part, need some explanations

**Review by KeerhemFR — CHANGES_REQUESTED** [review]

**Review by KeerhemFR — APPROVED** [review]
See no evil here

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**